docs/features/correlation.md

```diff
@@ -72,6 +72,10 @@ public void ConfigureServices(IServiceCollection services)
         // The header to look for in the HTTP request, and will be set in the HTTP response (default: X-Transaction-ID).
         options.Transaction.HeaderName = "X-Transaction-ID";
 
+        // The function that will generate the transaction ID, when the `.GenerateWhenNotSpecified` is set to `false` and the request doesn't contain the header.
+        // (default: new `Guid`).
+        options.Transaction.GenerateId = () => Guid.NewGuid().ToString();
```

Comment:
Maybe use something that differentiates from default such as :
```suggestion
        options.Transaction.GenerateId = () => $"Transaction-{Guid.NewGuid().ToString()}";
```

Reply:
Sure!